lib/galaxy/config/__init__.py

```diff
@@ -629,6 +630,7 @@ class GalaxyAppConfiguration(BaseAppConfiguration, CommonConfigurationMixin):
         "user_library_import_symlink_whitelist": "user_library_import_symlink_allowlist",
         "fetch_url_whitelist": "fetch_url_allowlist",
         "containers_resolvers_config_file": "container_resolvers_config_file",
+        "activation_email": "email_from",
```

Comment:
I think `activation_email` also needs to be added to `deprecated_options`

Reply:
Yes, good catch!

Reply:
This seems error prone, what about moving `deprecated_options` definition after `renamed_options`, and change it to something like:
```python
    deprecated_options = list(renamed_options.keys()) + [
        "database_file",
        "track_jobs_in_database",
    ]

Reply:
@nsoranzo Yes, it is error prone. I'll add this fix (thanks!), but I hope to replace this sometime in the near future: I've been working on a pet project for our config system which I hope to get into production shape reasonably soon (this summer, post-GCC, I hope) - it should take care of such awkwardness across the board.